jcl/source/common/JclSysUtils.pas

```diff
@@ -2380,7 +2380,7 @@ constructor TJclNumericFormat.Create;
   FSignChars[False] := '-';
   FSignChars[True] := '+';
   FPaddingChar := ' ';
-  FMultiplier := '�';
+  FMultiplier := '×';
```

Comment:
This should not have been changed, the encoding of JCL files must never be UTF-8 as it is not understood by early versions of Delphi/BCB